.github/workflows/build-lint-test.yml

```diff
@@ -0,0 +1,32 @@
+name: Build, Lint, and Test
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+jobs:
+  build-lint-test:
+    name: Build, Lint, and Test
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        node-version: [12.x]
```

Comment:
Only `12.x` here? Not 14 and 16 as well?

Reply:
fixed